brands/ghostery/mozconfig

```diff
@@ -3,6 +3,9 @@ ac_add_options --with-app-name=Ghostery
 ac_add_options --enable-official-branding
 ac_add_options --with-branding=browser/branding/ghostery
 ac_add_options --enable-update-channel=release
+ac_add_options --disable-verify-mar
+ac_add_options --disable-crashreporter
+ac_add_options MOZ_TELEMETRY_REPORTING=
```

Comment:
That is how it should be done? Not with `export`?

Reply:
or it should rather land in https://github.com/ghostery/user-agent-desktop/blob/b9c22830dd6e0cb30e405f0fdcaa0dc77b19994e/brands/ghostery/branding/configure.sh ?

Reply:
This a line I found from some mozilla configs: https://searchfox.org/mozilla-central/source/browser/config/mozconfigs/linux64/nightly-asan#27

Reply:
weird

Reply:
I'll test the different options to see if this is the best approach.

Reply:
I'm guessing that mozilla does not use `branding/configure.sh` as this is customised for release channels rather than for brandings.

Some context https://phabricator.services.mozilla.com/D30336